include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -377,6 +377,43 @@ def StaticRepeatOp : CalyxContainer<"static_repeat", [
 
 }
 
+def RepeatOp : CalyxContainer<"repeat", [
+    ControlLike
+  ]> {
+  let summary = "Calyx Dynamic Repeat";
+  let arguments = (ins
+    I32Attr:$count
+  );
+  let description = [{
+    The "calyx.repeat" operation represents the repeated execution of
+    the control within its region.
+    You must specify (statically) how many times to repeat the function. 
+    The "dynamic" part is that the body that you repeat can be dynamically timed, 
```

Comment:
Not: This is worded a bit weird/repetitively.

Maybe something closer to:

The "dynamic" part is the body of this control operation, which, unlike its static counterpart, does not need static timing.